### ISSUE TITLE
508 Issues

### ISSUE THREAD

**Issue by nickbristow** [18F/fec-cms#208]
https://beta.fec.gov/data/candidate/S6AZ00019/?cycle=2016&election_full=false

- [ ] Time period buttons, when changing with keyboard you lose focus on change

https://beta.fec.gov/registration-and-reporting/

- [x]  White text in the red "Need help" box fails color contrast, just barely 4.47:1
^^ this issue appears here as well https://beta.fec.gov/registration-and-reporting/essentials-house-and-senate-candidates-and-committees/

https://beta.fec.gov/data/committee/C00237198/
- [ ] Show total disbursements button says "show total receipts" (screen reader)

**Comment by noahmanger** [comment]
Thanks, Nick. 

Time period focus: those actually trigger a page reload, which is why focus is lost. Eventually we're going to refactor to handle more of this on the client side. Is that ok to leave as is for now?

On the color contrast, @jenniferthibault looks like we're using the "bright red" (`#dd3a00`) color here. Do you want to tweak that value to make it a little darker?

On the disbursements button, oops! We have a pending PR that actually changes the way the accordions work to follow the web design standards model, so this won't be relevant soon.

**Comment by jenniferthibault** [comment]
@noahmanger sure, let's make the combo #DC3B00 and white (#fff) instead

**Comment by noahmanger** [comment]
Got it. Thanks.

**Comment by nickbristow** [comment]
@noahmanger if you're planning on changing it then yeah, no sense coming up with two solutions.

**Comment by noahmanger** [comment]
Ok, fixed the contrast, and the other issues will be resolved soon. Closing.
